### DISCUSSION TITLE
What is the entry point for the cluster?

### DISCUSSION THREAD

**Discussion by AleksandrMalinovskiy** [autobase-tech/autobase#209] [Q&A]
For tests and training, I installed a cluster without load balancing[Type B].
What is the entry point for the cluster?
How to test a cluster? On which port of the master to connect and with what data?
Using pgAdmin I can not connect to the master on port 5432. Here is the postgresql.conf setting
listen_addresses = '*'
______
pg_hba.conf 
host all all all trust

P.s. sorry for bad english

**Comment by vitabaks** [comment]
If you have installed pgbouncer then as a rule it is port 6432

**Reply by vitabaks** [reply]
the VIP specified in the `cluster_vip` variable is usually used as a single entry point.

**Reply by AleksandrMalinovskiy** [reply]
Thanks
